### PR TITLE
fix: marketplace appsfilter component

### LINKED ISSUE #29432: Marketplace AppFilters component changed
### Description:

It seems that the `AppsFilter` component of Marketplace has had some UI changes which is making it look bad. I am not sure if it has been done purposely or it was by mistake, but in my opinion the previous UI looked much better and cleaner.

### Steps to reproduce:

1. Open your Rocket.chat application in browser
2. Go to Marketplace
3. Click on Explore

### Expected behavior:

issue_expected_behaviour

### Actual behavior:

https://github.com/RocketChat/Rocket.Chat/assets/68077049/7d00577e-c0da-4b15-ab34-701a6c8f3b9a

### Server Setup Information:

- Version of Rocket.Chat Server: `6.3.0-develop`
- Operating System: 
- Deployment Method: `gitpod`
- Number of Running Instances: `1`
- DB Replicaset Oplog: 
- NodeJS Version:` v14.21.3`
- MongoDB Version: `6.0.3 / wiredTiger (oplog Enabled)`

### PR DESCRIPTION
**Pull request by anikdhabal**
## Issue(s)

fix #29432

### CODE REVIEW

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# This configuration file was automatically generated by Gitpod.
```
Comment (debdutdeb) [inline .gitpod.yml]: Please remove this.

**Review by debdutdeb — CHANGES_REQUESTED** [review]

File: .gitpod.yml (hunk 1 above)
Comment (anikdhabal) [reply .gitpod.yml]: removed

**Comment by juliajforesti** [comment]
This PR #29443 fixed it :)

**Comment by anikdhabal** [comment]
> This PR #29443 fixed it :)

Oo okk

**Pull request closed without merging**